src/content/marina/marinaBroker.ts

```diff
@@ -410,32 +433,33 @@ export default class MarinaBroker extends Broker<keyof Marina> {
               changeCovenant?.template
             )
           );
+          // TODO: clarifiy the usage of isSpendableByMarina and understand
+          // how to set that value in case it's needed.
+
+          // const selectedAccount = selectAccount(this.selectedAccount)(this.state);
+          // const watchIdentity = await selectedAccount.getWatchIdentity(selectNetwork(this.state));
+
+          // const nextAddress = (await watchIdentity.getNextAddress()) as TaprootAddressInterface;
+          // const autoSpendableLeaf = (needsOfLeaf: ScriptInputsNeeds) =>
+          //   needsOfLeaf.sigs.length === 1 &&
+          //   !needsOfLeaf.needParameters &&
+          //   !needsOfLeaf.hasIntrospection;
+          // let isSpendableByMarina = Object.values(
+          //   analyzeTapscriptTree(nextAddress.taprootHashTree)
+          // ).some(autoSpendableLeaf);
+
+          // if (changeCovenant) {
+          //   const nextChangeAddress =
+          //     (await watchIdentity.getNextChangeAddress()) as TaprootAddressInterface;
+          //   isSpendableByMarina ||= Object.values(
+          //     analyzeTapscriptTree(nextChangeAddress.TaprootAddressInterface)
+          //   ).some(autoSpendableLeaf);
+          // }
+
+          // await this.store.dispatchAsync(
+          //   setIsSpendableByMarina(this.selectedAccount, isSpendableByMarina)
+          // );
```

Comment:
@louisinger help wanted on this:
I didn't get the meaning of `isSpenableByMarina`. If I'm not wrong it's used only for custom script accounts, but why? In theory, every custom account imported is indeed spendable by marina, isn't it?

I left the code commented since I'm not sure what to do with this logic, but it cannot stay here because it requires generating an address at import time (with no constructor args), which is not allowed anymore.